### PR TITLE
New Seeder for updating providers

### PR DESCRIPTION
**Pull request by AkhileshNegi**

### CODE REVIEW

File: lib/glific/partners.ex [hunk 1]
```diff
@@ -38,8 +38,10 @@ defmodule Glific.Partners do
 
   """
   @spec list_providers(map()) :: [%Provider{}, ...]
-  def list_providers(args \\ %{}),
-    do: Repo.list_filter(args, Provider, &Repo.opts_with_name/2, &filter_provider_with/2)
+  def list_providers(args \\ %{}) do
+    Repo.list_filter(args, Provider, &Repo.opts_with_name/2, &filter_provider_with/2)
+    |>Enum.reject(fn provider -> Enum.member?([3, 5, 8], provider.id) end)
```
Comment (pankaj-ag) [inline lib/glific/partners.ex]: let's use shortcodes here.

File: priv/repo/seeds/20201230073325_v0_7_0_alter_glific_data.exs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    update_existing_credentials()
+    delete_provider()
+  end
+
+  defp update_existing_credentials() do
+    credentials = Repo.all(Credential, skip_organization_id: true)
+
+    credentials
+    |> Enum.each(fn credential ->
+      if Enum.member?([3, 5], credential.provider_id) do
```
Comment (pankaj-ag) [inline priv/repo/seeds/20201230073325_v0_7_0_alter_glific_data.exs]: Same here?

File: priv/repo/seeds/20201230073325_v0_7_0_alter_glific_data.exs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        update_status(credential)
+      end
+    end)
+  end
+
+  defp update_status(credential) do
+    Repo.update!(Ecto.Changeset.change(credential, %{is_active: false}))
```
Comment (pankaj-ag) [inline priv/repo/seeds/20201230073325_v0_7_0_alter_glific_data.exs]: Can you put is_valid false as well?

**Review by pankaj-ag — CHANGES_REQUESTED** [review]

File: priv/repo/seeds/20201230073325_v0_7_0_alter_glific_data.exs (hunk 2 above)
Comment (AkhileshNegi) [reply priv/repo/seeds/20201230073325_v0_7_0_alter_glific_data.exs]: @pankaj-ag we are checking provider_id in credentials here. So for using shortcut we might need to preload here. Shall I do that?

**Review by pankaj-ag — APPROVED** [review]

**Pull request merged**